### DISCUSSION TITLE
Run as administrator not working

### DISCUSSION THREAD

**Discussion by ms4ndst** [Eugeny/tabby#5393] [General]
Have installed Tabby on my Windows 11 box and when I check the "Run as administrator" on a new profile the tab that opens closes immediately.  Couldn't find that there was a logfile.

edit: workaround is to install sudo for Windows

**Comment by JTDimino** [comment]
I am having the same issue but on a Windows 10 host...

**Comment by mflu** [comment]
I am having the same issue but on Windows 10

**Comment by 1oker** [comment]
I am having the same issue

**Comment by heartofrevel** [comment]
+1
